tube-player/README.md

```diff
@@ -37,22 +37,22 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
+- [C# Markup](xref:Uno.Extensions.Markup.Overview)
+- [Extensions](xref:Uno.Extensions.Overview)
+  - [MVUX](xref:Uno.Extensions.Mvux.Overview)
+    - [Pagination](xref:Uno.Extensions.Mvux.Advanced.Pagination)
+    - [FeedView](xref:Uno.Extensions.Mvux.FeedView)
       - NoneTemplate
       - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+    - [Refit](xref:Uno.Extensions.Http.Overview)
+    - [Region navigation](xref:Uno.Extensions.Navigation.Overview)
+    - [Serialization](xref:Uno.Extensions.Serialization.Overview)
 - [Figma plugin](https://aka.platform.uno/uno-figma)
   - C# Markup
   - Color overrides
-- [Color overrides](xref:uno.themes.material.getstarted)
-- [DSP import](xref:Uno.Material.DSP)
-- [Material design theme](xref:uno.themes.material.getstarted)
+- [Color overrides](xref:Uno.Themes.Material.GetStarted)
+- [DSP import](xref:Uno.Themes.Material.DSP)
+- [Material design theme](xref:Uno.Themes.Material.GetStarted)
 - [Toolkit](xref:Toolkit.GettingStarted)
   - [CardContentControl](xref:Toolkit.Controls.Card)
   - [NavigationBar](xref:Toolkit.Controls.NavigationBar)
```

Comment:
@vatsashah45 make sure to rebase your changes with latest master to resolve the conflicts, right now the content of README.md does not match the latest changes (https://github.com/unoplatform/workshops/pull/157/files)
![image](https://github.com/unoplatform/workshops/assets/16295702/4ad512b4-b7f3-4fd4-88e5-04e6bdcf3f29)